examples/01-filter/gradients.py

```diff
@@ -62,12 +63,11 @@ def gradients_to_dict(arr):
 keys = np.array(list(gradients.keys())).reshape(3, 3)
 
 p = pv.Plotter(shape=keys.shape)
-for i in range(keys.shape[0]):
-    for j in range(keys.shape[1]):
-        name = keys[i, j]
-        p.subplot(i, j)
-        p.add_mesh(mesh_g.contour(scalars=name), scalars=name, opacity=0.75)
-        p.add_mesh(mesh_g.outline(), color="k")
+for i, j in product(range(keys.shape[0]), range(keys.shape[1])):
+    name = keys[i, j]
```

Comment:
Even more obscure API but since `keys` is already a NumPy array I think we could do
```suggestion
for (i, j), name in np.ndenumerate(keys):
```
(also untested!).